tutorials/llm/llama-3/pruning-distillation/README.rst

```diff
@@ -1,18 +1,26 @@
-Llama 3.1 WikiText Pruning and Distillation with NeMo Framework
+Llama 3.1 Pruning and Distillation with NeMo Framework
 =======================================================================================
 
 `Llama 3.1 <https://blogs.nvidia.com/blog/meta-llama3-inference-acceleration/>`_ are open-source large language models by Meta that deliver state-of-the-art performance on popular industry benchmarks. They have been pretrained on over 15 trillion tokens, and support a 128K token context length. They are available in three sizes, 8B, 70B, and 405B, and each size has two variants—base pretrained and instruction tuned.
```

Comment:
revise paragraph

LLama 3.1 models, developed by Meta, are open-source large language models that deliver state-of-the-art performance on popular industry benchmarks. Pretrained on over 15 trillion tokens, they support a 128K token context length. These models are available in three sizes: 8B, 70B, and 405B. Each size offers two variants: base pretrained and instruction tuned.